Backend/dev/test-tool/specs/ny-rn-consumer.yaml

```
@@ -12,16 +12,23 @@ source:
+  - { name: backend, port: 8013, protocol: http, registryKey: rider-app }
+  - { name: mock,    port: 8080, protocol: http, registryKey: mock-server }
 
-adbReverse: [metro, backend, mock]
+adbReverse: [metro, caddy, backend, mock]
 
 tools: [node22, java17, yarn, adb, coreutils, gawk]
 
 env:
   ANDROID_HOME: "${env.ANDROID_HOME}"
 
+env:
```

YAMLlint (1.38.0)

[error] duplication of key "env" in mapping

(key-duplicates)